client/layout/guided-tours/config-elements.js

```diff
@@ -141,8 +141,8 @@ export class Step extends Component {
 	 * Needed for analytics, since GT is selector-driven
 	 */
 	start() {
-		const { start, tour, tourVersion } = this.context;
-		start( { tour, tourVersion } );
+		const { start, tour, tourVersion, step } = this.context;
+		start( { tour, tourVersion, step } );
```

Comment:
I think our convention so far has been to alphabetically order these. Can you re-order?